src/components/MapView/Download/report.tsx

```diff
@@ -0,0 +1,172 @@
+import React from 'react';
+import {
+  Button,
+  createStyles,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  IconButton,
+  Theme,
+  useTheme,
+  WithStyles,
+  withStyles,
+} from '@material-ui/core';
+import { useSelector } from 'react-redux';
+import { ArrowBack } from '@material-ui/icons';
+import { PDFViewer, PDFDownloadLink } from '@react-pdf/renderer';
+import { useLocation } from 'react-router-dom';
+import { useSafeTranslation } from '../../../i18n';
+import { mapSelector } from '../../../context/mapStateSlice/selectors';
+import ReportDoc from './reportDoc';
+import {
+  analysisResultSelector,
+  getCurrentData,
+} from '../../../context/analysisResultStateSlice';
+import { ReportType } from '../utils';
+
+function Report({ classes, open, reportType, handleClose }: ReportProps) {
+  const theme = useTheme();
+  const { t } = useSafeTranslation();
+  const [mapImage, setMapImage] = React.useState<string>('');
+  const selectedMap = useSelector(mapSelector);
+  const analysisData = useSelector(getCurrentData);
+  const analysisResult = useSelector(analysisResultSelector);
+  const location = useLocation();
+  const dateQuery = location.search
```

Comment:
It might be cleaner to add the date as a return info in `analysisData` and get it from there instead to make sure that we stay consistent. What do you think?